sarahricker/thinkery
# [THNK-41] calendar block

## JIRA Ticket

[THNK-41](https://wpengine.atlassian.net/browse/THNK-41)

## Description

A few sentences describing the overall goals of the pull request's commits.

PR makes use of [Modern Tribe Events Calendar Pro](https://theeventscalendar.com/), the [Events Calendar Category Colors](https://wordpress.org/plugins/the-events-calendar-category-colors/) plugin + many template + style overrides to get us as close to the custom calendar block design as possible. 

## Impacted Areas in Application

List general components of the site that this PR will affect:

* Calendar + Events throughout the site (notably Day view + Mini Calendar widget thus far)

## Deploy Notes

Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

![Screen Shot 2019-11-25 at 5 21 48 PM](https://user-images.githubusercontent.com/26469099/69586425-29e4bc00-0fa8-11ea-89eb-f6376743687f.png)

## Steps to Test or Reproduce

1. pull down branch
2. ensure you have both plugins above
3. Setup plugins (see deploy notes)
4. Add some test events

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/4Z9jjKGJVM9Ck8E4cN/giphy.gif)
